packages/firebase/index.d.ts

```diff
@@ -5473,6 +5473,57 @@ declare namespace firebase.storage {
      *     including if the object did not exist.
      */
     updateMetadata(metadata: firebase.storage.SettableMetadata): Promise<any>;
+    /**
+     * List items and sub-directories within this directory.
+     *
+     * "/" is treated as a path delimiter. Firebase storage does not support
+     * invalid object path that ends with "/" or contains two consecutive "//".
+     * All invalid objects in GCS will be filtered.
+     *
+     * @param options.maxResults If set, limits the total number of `prefixes`
+     *      and `items` to return.
+     * @param options.pageToken The `nextPageToken` from a previous call to
+     *      list(). If provided, listing is resumed from the previous position.
+     * @return A promise that resolves with the items and sub-directories.
+     *      `prefixes` contains references to sub-directories and `items`
+     *      contains references to objects in this folder. `nextPageToken`
+     *      can be passed as `options.pageToken` to get the rest of results.
+     */
+    list(options?: ListOptions): Promise<ListResult>;
+  }
+
+  /**
+   * Result returned by list()
+   */
+  interface ListResult {
+    /**
+     * References to sub-directories. You can call list() on them to get its contents.
+     */
+    prefixes: Reference[];
+    /**
+     * Objects in this directory. You can call getMetadate() and getDownloadUrl() on them.
+     */
+    items: Reference[];
+    /**
+     * If set, there might be more results for this list. Use this token to resume the list.
+     */
+    nextPageToken: string | null;
+  }
+
+  /**
+   * The options list() accepts
+   */
+  interface ListOptions {
+    /**
+     * The maximum number of results returned.
+     */
+    maxResults?: number | null;
+    /**
+     * A pageToken returned from a previous call to list().
+     * If set, resumes list from the previous location. prefixes and items already listed
```

Comment:
I would remove the second sentence. If not, s/prefixes/Prefixes/

Reply:
Removed